Cpp2IL.Core/CorePlugin/AttributeAnalysisProcessingLayer.cs

```diff
@@ -7,7 +7,7 @@ namespace Cpp2IL.Core.CorePlugin;
 
 public class AttributeAnalysisProcessingLayer : Cpp2IlProcessingLayer
 {
-    public override string Name => "CustomAttribute Analyser";
+    public override string Name => "CustomAttribute Analyzer";
```

Comment:
Not sure this is necessary, you're touching a file to change a UK spelling to a US one.

Reply:
Consistency. The next line uses the US spelling.